CHANGELOG.md

```diff
@@ -1,6 +1,26 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [0.5.0-beta] - 2022-03-05
+### Added
+  * Added support for Linux ID-mapped mounts (shiftfs alternative in kernels >= 5.12).
+  * Added support for ARM hosts.
+  * Added support for running buildx/buildkit inside Sysbox containers.
+  * Added support for running Rancher RKE2 and Mirantis K0s inside Sysbox containers.
+  * Added configs to disable trapping chown and xattr* syscalls (improves performance but may reduce functionality).
+  * Added config to strictly honor container capabilities from higher level container manager.
+  * Added support for per-container configs via `SYSBOX_*` env vars.
+  * Improved performance of Sysbox's syscall interception code.
+  * Improved the way Sysbox releases the seccomp-fd handles for intercept syscalls (kernels >= 5.8).
+  * Update to golang 1.16.
+  * Fixed segfault when building Docker image inside Sysbox container (issue 484).
+  * Fixed segfault when running python pip install inside nested sysbox container (issue 485).
+  * Fixed issue with running KinD inside a Sysbox container (issue 415).
+  * Fixed problem with shiftfs mounts on Kubernetes persistent volumes (issue 431).
+
```

Comment:
@ctalledo, @rodnymolina I think you guys forgot to mention:

* Allow alternative docker data-dir inside a Sysbox container by reading `/etc/docke/daemon.json`

Reply:
Done adding this.